templates/examples/Selection.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import CenteredSelection from '../CenteredSelection'
+import FilteredSelection from '../FilteredSelection'
+import Selection from '../Selection'
+import Wrapper from '../Wrapper'
+import DemoIcon from './DemoIcon'
+import { TEMPLATE } from '../../Showroom/variationTypes'
+
+const optionsData = [
+  {key: 'lor', label: 'Lorem'},
+  {key: 'ip', label: 'Ipsum'}
+]
+
+export default {
+  title: 'Selection',
+  variations: [
+    {
+      title: 'Selection',
+      require: 'import Selection from \'@klarna/ui/templates/Selection\'',
+      type: TEMPLATE,
+
+      examples: {
+        Regular: {
+          inline: <Selection
+            title='What option do you prefer?'
+            summary='Chia williamsburg subway tile vaporware, live-edge kinfolk cardigan prism deep v retro seitan.'
+            options={optionsData}
+            value='ip'
+            onSelect={(key) => console.log('selected ' + key)}
+          />,
+
+          wrapper: <Wrapper>
+            <Selection
+              title='What option do you prefer?'
+              summary='Chia williamsburg subway tile vaporware, live-edge kinfolk cardigan prism deep v retro seitan.'
+              options={optionsData}
+              value='ip'
+              onSelect={(key) => console.log('selected ' + key)}
+            />
+          </Wrapper>
+        }
+      }
+    },
+
+    {
+      title: 'FilteredSelection',
+      require: 'import FilteredSelection from \'@klarna/ui/templates/FilteredSelection\'',
+      type: TEMPLATE,
+
+      examples: {
+        Regular: {
+          inline: <FilteredSelection
+            title='Select your flavor'
+            summary='Photo booth distillery man bun, bitters stumptown freegan cliche cronut green juice.'
+            label='Start typing a flavor'
+            options={[
+              {key: 'chocolate', label: 'Chocolate'},
+              {key: 'vanilla', label: 'Vanilla'},
+              {key: 'raspberry', label: 'Raspberry'},
+              {key: 'tiramisu', label: 'Tiramisu'},
+              {key: 'mascarpone', label: 'Mascarpone'},
+              {key: 'banana-split', label: 'Banana Split'},
+              {key: 'anana', label: 'Anana'}
```

Comment:
I wonder why we need to do these key/label objects... I guess it's because JS doesn't have proper pairs. I'm planning to add 'em to ion BTW